### ISSUE TITLE
Update Templates and Issues

### ISSUE THREAD

**Issue by Snugug** [punchcard-cms/content-types#120]
Blocked on https://github.com/punchcard-cms/punchcard/issues/479

```gherkin
Feature: Project Maintenance
  As a Core Developer
  I want to have a consistent project maintenance experience with Punchcard
  So that I don't have to switch mental models when doing project maintenance

  Scenario: Templates
    Given that I am creating issues or PRs in Content Types
     When a template is used to pre-fill the templates
     Then it should be identical to the templates used in Punchcard

  Scenario: Labels
    Given that I am creating or editing issues or PRs in Content Types
     When I want to label my issues to categorize them
     Then the labels should be identical to the ones used in Punchcard
      But the Section labels should be unique to Content Types
      And Reparo should be set up to manage them automatically
```

---
**Project Maintenance**
- [ ] Templates
- [ ] Labels

**Comment by scottnath** [comment]
@Snugug 

```
  Scenario: Milestones
    Given that I am creating milestones in Content Types
     When an issue requires a milestone
     Then it should be identical to the milestones used in Punchcard
```

**Comment by Snugug** [comment]
There's no way to sync milestones, so that's out of scope

**Comment by scottnath** [comment]
boooo
